colossalai/fx/passes/algorithms/linearize.py

```diff
@@ -1,6 +1,25 @@
-from typing import List
+from typing import List, Any
 from torch.fx import GraphModule, Node
 
+# List of target name that could be seen as common node
```

Comment:
Include the definition of `common node` in the documentation for future reference.

Reply:
okay, I will add the definition